inst/templates/bs4_book.html

```diff
@@ -122,6 +122,25 @@ <h2>On this page</h2>
     document.getElementsByTagName("head")[0].appendChild(script);
   })();
 </script>
+<!-- https://stackoverflow.com/a/49687594 -->
+<script type="text/x-mathjax-config">
+const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
+for (let popover of popovers){
+  const div = document.createElement('div');
+  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
+  div.innerHTML = popover.getAttribute('data-content');
+
+  // Will this work with TeX on its own line?
```

Comment:
Just to be sure: what does this mean ? did you test ?

Reply:
Now tested, I think I forgot to remove the comment after supporting multiline footnotes in `bs4_book()`.